src/templates/layouts/pressRelease/index.test.tsx

```diff
@@ -156,40 +156,27 @@ describe('<pressRelease> with valid data', () => {
     const ext = screen.getByTestId('phone-3')
     const fax = screen.getByTestId('phone-4')
     expect(voice).toBeInTheDocument()
-    expect(voice).toHaveAttribute('contact', data.contacts[0].numbers[0].number)
```

Comment:
Since this is switching to the shared component, we no longer need to test all of this logic here.